### ISSUE TITLE
Can't install Arc-Menu

### ISSUE THREAD

**Issue by pstnta** [LinxGem33/Arc-Menu#139]
Hi,

**Information about your Problem.**

I tested all this in a VM, I was using basic gnome on ubuntu server (only installing gnome-shell & gnome-tweak-tool) but it doesn't work. I tested it on Ubuntu Gnome and it works just fine, so I'm guessing the basic gnome is missing dependencies. I did install **gir1.2-gmenu-3.0 and gnome-shell-extensions** as was recommended in previous threads but the issue still persists. When I click the ON button on gnome extensions, it just changes to a red ERROR button.

**Distro info**

Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.3 LTS
Release:	16.04
Codename:	xenial

**gnome-shell --version**

GNOME Shell 3.18.5

**journalctl /usr/bin/gnome-shell -f -o cat**
When I run this command, and then try to install arc menu. All I get is the following, 
GNOME Shell started at Thu Aug 31 2017 05:16:21 GMT-0700 (PDT)
any other place I can look for a log ?

thanks!

**Comment by lexruee** [comment]
@pstnta @pmorillo @emjimadhu
The main problem is that a lot of users try to install a GNOME system from scratch and then they end up with a broken GNOME system and blame the developers for their ignorance. The GNOME 3 Desktop is a complex piece of software with lot of dependencies!

The root cause is that novice users forget to install all the optional GNOME dependencies. This applies to Ubuntu users, Debian users, and some Arch Linux users. I mean there is a good reason why GNOME distributions such as Ubuntu GNOME, Debian or Antergos exist.

Anyways, I fired up two minimal Ubuntu virtual machines (**16.04, 17.04**) to solve this problem **for once and for all**. The starting point were the Netboot iso from here: 

- http://cdimage.ubuntu.com/netboot/xenial/
- http://cdimage.ubuntu.com/netboot/zesty/

To install a minimal, usable GNOME System you need to install these packages here:
```
sudo apt install -y gnome-shell gnome-tweak-tool gnome-menus firefox chrome-gnome-shell
sudo reboot
```

After that please visit the website https://extensions.gnome.org/extension/1228/arc-menu/ and install Arc-Menu from there!

In summary, to install Arc-Menu via firefox and [extensions.gnome.org](https://extensions.gnome.org/), you need to install the following packages:
- gnome-tweak-tool
- gnome-menus
- firefox
- chrome-gnome-shell

**Comment by pstnta** [comment]
Thanks man! gnome-menus was missing. I really appreciate it, your extension is really awesome!

**Comment by lexruee** [comment]
@pstnta 
Good to hear that this solved your issue. I think it is okay that I close this issue for you.

**PS:**
Personally, I never install a minimal GNOME system solely based on the `gnome-shell` package as it lacks almost all necessary GNOME dependencies. A better approach to install a minimal GNOME system is using the meta-package `gnome-core`, which contains the core components of the GNOME Desktop environment.

**Comment by pstnta** [comment]
I didn't know about gnome-core, ill check it out, thanks
